### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewOutput_ScreenReader: Jaws does not announce "cannot edit in read only editor", when we type something in editor.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52190]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: 18.0.5038

Repro Steps:
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Output.
3. Select Log(Main) or any of the items from the combobox after Terminal tab item.

**Actual:**
 Jaws does not announce "cannot edit in read only editor", when we type something in editor while a notification pop ups "cannot edit in read only editor" when we enter any key.

**Expected:**
Jaws should read "cannot edit in read only editor" when user enters any key.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know that the editor is read only.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Output Jaws Read Only Editor.pptx](https://github.com/Microsoft/vscode/files/2111039/A11y.VSCode.Output.Jaws.Read.Only.Editor.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
We do not support jaws
